cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/error/mapper/ClouderaManagerStorageErrorMapper.java

```diff
@@ -76,11 +76,11 @@ private String awsError(CloudStorage cloudStorage) {
                         "This problem usually occurs due to cloud storage permission misconfiguration. " +
                         "Services on the cluster are using Data Access Role (%s) and Ranger Audit Role (%s) to write to the Ranger Audit location (%s), " +
                         "therefore please verify that these roles have write access to this location. " +
-                        "During Data Lake cluster creation, CDP Control Plane attaches Data Access Instance Profile (%s) to the IDBroker VM. " +
-                        "IDBroker will then use it to assume the Data Access Role and Ranger Audit Role, therefore Data Access Instance Profile (%s) " +
+                        "During Data Lake cluster creation, CDP Control Plane attaches Assumer Instance Profile (%s) to the IDBroker VM. " +
```

Comment:
VM or Virtual Machine which one is nicer ?

Reply:
Maybe VM is not obvious for everybody. I have updated to Virtual Machine.